public/base-locales/pl.json

```diff
@@ -1,50 +1,50 @@
 {
-  "login": "Login",
+  "login": "Zaloguj się",
   "click_once": "Kliknij 'Autoryzuj' tylko raz",
   "save": "Zapisz",
   "reset": "Zresestuj",
   "close": "Zamknij",
-  "reset_filters": "Zresetuj filtry",
+  "reset_filters": "Reset ustawień",
   "pokemon_filters": "Ustawienia filtrów Pokemonów",
   "gyms_filters": "Ustawienia filtrów gymów",
   "pokestops_filters": "Ustawienia filtrów Pokestopów",
   "nests_filters": "Ustawienia filtrów nestów",
   "help": "Pomoc",
   "apply_to_all": "Zastosuj do wszystkich",
-  "disable_all": "Wyłącz wszystkie",
-  "enable_all": "Włącz wszystkie",
-  "enabled": "Włączony",
+  "disable_all": "Dezaktywuj wszystkie",
+  "enable_all": "Aktywuj wszystkie",
+  "enabled": "Aktywuj",
```

Comment:
Why? It sounds weird.

Reply:
imo it's more describing and fits better